README.md

```diff
@@ -291,6 +291,9 @@ The link holds all necessary information to restore vector field state (this is
 If you'd like to record a video with a vector tool, please refer to [instructions in this
 file](https://github.com/anvaka/fieldplay/blob/master/ScreenRecording.md).
 
+## Auto Mode
```

Comment:
This is not a core feature, so maybe we just document it here in the Readme rather than in the app itself.